src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -283,6 +283,100 @@ namespace Sapphire::Network::Packets::Server
     char unk2[0x4]; // This has probs something to do with the support desk (inquiry id?)
   };
 
+  struct FFFXIVIpcMarketBoardItemListingCount : FFXIVIpcBasePacket< MarketBoardItemListingCount >
+  {
+    uint32_t itemCatalogId;
+    uint32_t unknown1; // does some shit if nonzero
+    uint16_t requestId;
+    uint16_t quantity; // high/low u8s read separately?
+    uint32_t unknown3;
+  };
+
+  struct FFXIVIpcMarketBoardItemListing : FFXIVIpcBasePacket< MarketBoardItemListing >
+  {
+    struct itemListing // 152 bytes each
+    {
+      uint32_t unknown; // Changes if multiple packets are sent for a single item
+      uint8_t padding;
+      uint16_t unknown1;
+      uint8_t padding1;
+      struct unknown2
```

Comment:
is there a particular reason that you have this defined as a structure?

Reply:
It always contains the same data looped 1-3 times, can't figure out what it is though